### ISSUE TITLE
Players can't connect to the server cause is saying that "The player is already connected"

### ISSUE THREAD

**Issue by Xenoxis** [cuberite/cuberite#4609]
Client version: 1.12.2
Server OS: Ubuntu server 16.04 i386
Cuberite Commit id: 318d31dc394da9faea11544ad6e485f1b5eabbdb

### Expected behavior
Login to the server and start playing

### Actual behavior
When the player try to connect, it say that the player is already connected, which is false.
Currently the online-mode on my server is false, don't know if can be related

### Steps to reproduce the behavior
I know this is reproducible, i don't know how

**Comment by Xenoxis** [comment]
I think yes, everyone with who this happened where connected but they didn't kicked out by the server, one disconnected for a while and when he tried to connect he was kicked out because apparently he was already logged in
And yes I can confirm if you try to kick the "phantom" player which is already logged in, the server say that he was successfully kicked out, but the real player cannot still connect

**Comment by antgame11** [comment]
Still looking for a fix for this.

**Comment by MightyFilipns** [comment]
Could you try with the experimental branch? Sometimes, if a player disconnects during login, the server will still consider the player connected and therefore will not allow the player to rejoin.

**Comment by antgame11** [comment]
> Have you tried with the experimental branch? Sometimes, if a player disconnects during login, the server will still consider the player connected and therefore will not allow the player to rejoin.

yeah, I've tried just about everything. Could we talk on discord?

**Comment by antgame11** [comment]
> > > Have you tried with the experimental branch? Sometimes, if a player disconnects during login, the server will still consider the player connected and therefore will not allow the player to rejoin.
> > 
> > 
> > yeah, I've tried just about everything. Could we talk on discord?
> 
> I rather keep the discussion here. So if some in the future runs into this issue can the find this thread and fix themselves instead of creating a new issue. What branch of cuberite are you using? Which client version are you using to connect?
> 
> Try with the experimental branch using 1.21.4 as the client. If that fails send here the server log.

Ok, so I tried that but the server in question is a 1.12.2 server. I could never get new versions to work on the experimental branch, but, I just got Gemini to make a fix for me. It destroys the old client whenever a new client with the same username joins, check it out here.

[https://github.com/antgame11/cuberite/commit/b7de542ff69d6eedacbb03e82a4f777751578e24](https://github.com/antgame11/cuberite/commit/b7de542ff69d6eedacbb03e82a4f777751578e24)

it's not optimal but the server im using is behind bungee so it's fine for me

**Comment by MightyFilipns** [comment]
You should be able to set AllowMultiLogin=1 in the settings.ini file.
If you just clone and build experimental, you must use either 1.21.4 or 1.21.6; otherwise, you will spawn in an empty world.
To add support for other versions, you must add the blocks.json and registry.json files to the Protocol/VERSION_NAME/ folder

Also, files for tags are not currently in the repo, but must be added from the vanilla data gen.
